### PR TITLE
Buffing and polishing the Project Card in Discover and Make

### LINKED ISSUE #831: UI Polish - Project Card
All project cards
- [x] Avatar section has too much vertical space
- [x] Line space within two-line Title
- [x] Does userame need to be blue? (with icon there now) check twitter instagram facebook?
- [x] Should icon be gray?
- [x] no bleed on thumbnail

Project card in Make
- [x] Project Title is not vertically centered
- [x] More Actions icon can be a touch smaller perhaps
- [x] The Share/Delete lines in the More Actions modal are not vertically centered.

### PR DESCRIPTION
**Pull request by vazquez**
Fixes #831

### CODE REVIEW

**Comment by xmatthewx** [comment]
In case I don't have time to build the project today, can you post screenshots of the UI work? Or is that a pain in the rear?

**Comment by vazquez** [comment]
For you, anything.

screenshot_20151120-110951

File: src/components/card/card.less [hunk 1]
```diff
@@ -11,54 +11,62 @@
...
       display: block;
     }
   }
 
   .thumbnail {
-    padding: 6px 0 0 0;
+    padding: 0;
     height: 0;
-    padding-bottom: 61.875%;
+    padding-bottom: 64.875%;
```
Comment (xmatthewx) [inline src/components/card/card.less]: What's this oddly huge padding for?

**Comment by xmatthewx** [comment]
Subtle, but _much_ better looking. One question in the PR. Otherwise R+ :+1:

File: src/components/card/card.less (hunk 1 above)
Comment (vazquez) [inline src/components/card/card.less]: Padding between the thumbnail and the text. Not sure why it was set to percentage, so I thought it best to keep it the same value type.

Comment (xmatthewx) [inline src/components/card/card.less]: Seems like an odd hack added for image loading trouble here: https://github.com/mozilla/webmaker-core/commit/c44b55cef33a3db2e50f4821c8dcad26b397c354

Did you intend to change it by 3%? Was that to increase space below thumb and above icon and title?

Comment (vazquez) [inline src/components/card/card.less]: Yes, the 3% increase was intended for the reason you say above. I can check just a normal px or rem value and see what/if it breaks.

**Comment by xmatthewx** [comment]
Totally R+ now :+1:

**Comment by vazquez** [comment]
Can I merge? :dancer:

File: src/components/modal-switch/modal-switch.less [hunk 2]
```diff
@@ -13,6 +13,7 @@
       border: 0;
       border-bottom: 1px solid @heatherGrey;
       text-align: left;
+      line-height: 22px;
```
Comment (alanmoo) [inline src/components/modal-switch/modal-switch.less:16]: There isn't a way to make this unitless, is there?

Comment (vazquez) [inline src/components/modal-switch/modal-switch.less:16]: No :disappointed:

Comment (alanmoo) [inline src/components/modal-switch/modal-switch.less:16]: Oh well. Sometimes you just have to kneel at the mercy of CSS.

**Pull request merged**